### DISCUSSION TITLE
Não perder alterações feitas ao atualizar sistema

### DISCUSSION THREAD

**Discussion by guskid** [RamonSilva20/mapos#1605] [Ideas]
Saudações aos desenvolvedores e participantes. 
Venho utilizando o MAPOS a um bom tempo já e sempre faço alterações que se adaptam as necessidades da minha empresa. 
Tenho uma Gráfica e Comunicação Visual e alguns textos como "Defeito" , "Laudo técnico" , "Garantia" e alguns outros texto não são necessários, então eu faço alterações diretamente nos códigos dos arquivos porém ao atualizar o sistema sempre é perdido.
Minha dúvida é se tem como fazer alguma coisa para que quando tenha uma nova atualização do sistema, não seja perdido essas alterações? 
Uma ideia seria em ter como fazer essas alterações em um tipo de configuração dentro do próprio MAPOS. 
Algo onde a própria pessoa preenchesse os textos como esses que descrevi acima, os status dos pedidos entre outros textos que são "fixos".
Aqui, por exemplo, eu sempre preciso ter alguns status padrões como, "orçamento", "Em andamento", "Em produção", "Aguardando Retirada" e "Finalizado"...

Vou anexar um exemplo de como fica a impressão de OS que envio como orçamento aos meus clientes.

Obrigado a todos pelo tempo que passou lendo e ajudando. 
Um abraço!
![image](https://user-images.githubusercontent.com/61772592/148487771-4e1035a9-b225-4046-8b35-02ca94d75085.png)

**Comment by Pr3d4dor** [comment]
@guskid O único jeito de fazer isso, como as customizações são suas mesmo, é usar o GIT para fazer esse controle localmente.

**Reply by Pr3d4dor** [reply]
Você pode fazer um fork contendo as suas modificações e ir atualizando ele com as atualizações que saírem, sem perder as suas próprias.

**Reply by guskid** [reply]
Valeu @Pr3d4dor !
Vou pesquisar aqui como fazer isso. 
Se eu não encontrar posso pedir seu auxilio aqui? 
Obrigado!
